kolibri/plugins/coach/assets/src/composables/useQuizCreation.js

```diff
@@ -195,8 +200,8 @@ export default function useQuizCreation(DEBUG = false) {
    * @affects _channels - Calls _fetchChannels to bootstrap the list of needed channels
    * Adds a new section to the quiz and sets the activeSectionID to it, preparing the module for
    * use */
-  function initializeQuiz() {
-    set(_quiz, objectWithDefaults({}, Quiz));
+  function initializeQuiz(collection) {
```

Comment:
Could you add a `@param {string} collection - The collection (aka current class ID) to associate the exam with` to the doc string above here?